### DISCUSSION TITLE
Lack of code files?

### DISCUSSION THREAD

**Discussion by ghost** [jacobranz/PasswordManager#23] [Q&A] [answered]
Hi, I was just wondering if there were more code files that weren't public inside the repository. Looks like there's only a SQL database.

**Comment by jacobranz** [comment, marked as answer]
Hey @dishb, yes I do have more code mainly hosted in other branches at the moment. The main branch I am currently working off of is the gui_test branch. Where I am making efforts to develop the graphical logic to display the contents of the database as well as other features.

**Reply by jacobranz** [reply]
The initial_test branch has the code for the terminal version. Which is more complete in terms of the users ability to actually store, view, and edit credentials. Feel free to look around and propose changes. The gui_test branch is my current working order though.

**Reply by ghost** [reply]
Ah okay, didn't check those branches!
